web/src/main/java/pl/edu/agh/controller/UserControllerImpl.java

```diff
@@ -0,0 +1,37 @@
+package pl.edu.agh.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import pl.edu.agh.converter.UserConverter;
+import pl.edu.agh.dto.UserDTO;
+import pl.edu.agh.repository.UserRepository;
+
+@Controller
+public class UserControllerImpl implements UserController {
+    private final UserRepository userRepository;
+
+    private UserControllerImpl(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @Override
+    public ResponseEntity<List<UserDTO>> getUsers() {
+        List<UserDTO> userDTOS = userRepository.findAll().stream()
```

Comment:
`userDTOs` chyba byloby lepiej

Reply:
liczyłem na innego rodzaju feedback, ale to się ceni

Reply:
dobra, pomyliłem branche z tamtym commitem, dzięki